src/launcher/util.ts

```diff
@@ -14,9 +14,49 @@ export async function exists(path: string): Promise<Boolean> {
     }
 }
 
+function getProxy() {
+    const parsedUrl = new URL(proxyExists());
+    return {
+        port: parsedUrl.port,
+        host: parsedUrl.hostname,
+        username: parsedUrl.username,
+        password: parsedUrl.password
+    };
+}
+
+function proxyExists() {
+    return process.env.HTTPS_PROXY || process.env.https_proxy;
+}
```

Comment:
How about process.env.HTTP_PROXY/http_proxy?

Reply:
We are making an https request to get resolve the runtime version and download it, so I figured we would need to only care about HTTPS_PROXY being set.

Reply:
gotcha